examples/docusaurus-2/blog/2019-05-29-hello-world.md

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
I dont think this file should be removed.

But lets revisit this PR once we have v1 completely disabled for production deployments.